x/msgfees/types/fee.go

```diff
@@ -9,10 +9,17 @@ const (
 )
 
 // SplitCoinByBips returns split to recipient and fee module based on basis points for recipient
+// keeping this method, even though bips is being hardcoded to 100bips to recipient for now(12/14/2022)
 func SplitCoinByBips(coin sdk.Coin, bips uint32) (recipientCoin sdk.Coin, feePayoutCoin sdk.Coin, err error) {
 	if bips > 10_000 {
 		return recipientCoin, feePayoutCoin, ErrInvalidBipsValue.Wrapf("invalid: %v", bips)
 	}
+	// nothing to calculate if recipient gets 100 bips, short circuit
```

Comment:
10,000 bips is 100% ... instead of bips in this comment it seems like you wanted percentage

Reply:
:doh: correct, changed comment.